gaphor/storage/parser.py

```diff
@@ -311,7 +311,12 @@ def parse_generator(filename, loader):
     parser.setFeature(handler.feature_namespaces, 1)
     parser.setContentHandler(loader)
 
-    yield from parse_file(filename, parser)
+    try:
+        # returns only a progress percentage
+        yield from parse_file(filename, parser)
+    except UnicodeDecodeError:
+        # Fall back on default encoding
+        yield from parse_file(filename, parser, encoding=None)
```

Comment:
NB. This except block should allow old models, saved and loaded on windows, to load.